src/profile/Signup.tsx

```diff
@@ -1,5 +1,5 @@
 // hooks import
-import { FormEvent, useState } from "react";
+import { FormEvent, useEffect, useState } from "react";
```

Comment:
I think this useEffect import snuck in here by accident ... doesn't seem to be used in the rest of the code.